guides/spin/canary-configs/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: single
+title:  "Manage Canary Configs"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+## Overview
+
+`spin` can manage the whole lifecycle of your canary configs:
+
+```bash
+$ spin canary canary-configs
+Usage:
+   canary canary-config [command]
+
+Aliases:
+  canary-config, canary-configs, cc
+
+Available Commands:
+  delete      Delete the provided canary config
+  get         Get the canary config with the provided id
+  list        List the canary configs
+  retro       Retro the provided canary config
+  save        Save the provided canary config
+
+Flags:
+  -h, --help   help for canary-config
+
+Global Flags:
+      --config string            path to config file (default $HOME/.spin/config)
+      --default-headers string   configure default headers for gate client as comma separated list (e.g. key1=value1,key2=value2)
+      --gate-endpoint string     Gate (API server) endpoint (default http://localhost:8084)
+  -k, --insecure                 ignore certificate errors
+      --no-color                 disable color (default true)
+      --output string            configure output formatting
+  -q, --quiet                    squelch non-essential output
+
+Use " canary canary-config [command] --help" for more information about a command.
+```
+
+The following assumes Spinnaker is running and Gate is listening on `http://localhost:8084`. If
+gate is running elsewhere, you can set the Gate endpoint with the global `--gate-endpoint` flag.
+
+## Managing Your Canary Config's Lifecycle
+
+### Create and update canary configs with `save`
+
+```bash
+$ spin canary canary-configs save --file <path to canary config json>
+```
+
+Note that `save` accepts the canary config in JSON format. You can quickly export an existing
+canary config into a valid argument to the `--file` flag by using the `get` command.
+
+### List canary configs with `list`
+
+```bash
+spin canary canary-config list
+
+[
+...
+{
+  "id": "canaryConfigId",
+  "name": "canaryConfigName"
+  ...
+},
+...
+]
+
+```
+
+### Retrieve a single canary config with `get`
+
+```bash
+spin canary canary-config get canaryConfigId
```

Comment:
* The `save` version above is `canary-configs` (plural). This one is singular
* Is this different from the `get` version mentioned in that same place? That one takes a `--file`, but this one takes `canaryConfigId`, but are we doing the same thing here?

Reply:
> That one takes a --file

It's the same `get. The `--file` belongs to `save`. That sentence was explaining how to use `get` as an argument to the `--file` param of `save.